staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -1370,8 +1370,9 @@ type InterfaceSRIOV struct{}
 // Deprecated: Removed in v1.3
 type DeprecatedInterfaceMacvtap struct{}
 
-// InterfacePasst connects to a given network.
-type InterfacePasst struct{}
+// DeprecatedInterfacePasst connects to a given network.
+// Deprecated: Removed in v1.3
```

Comment:
Same, please use the same format as macvtap above.

Reply:
Done